### ISSUE TITLE
Chrome extension support

### ISSUE THREAD

**Issue by Nightbr** [ionic-team/ionic-storage#152]
Hello,

we are building a chromeApp (or ChromeExtension) from an ionic app and it seems we can't use window.localstorage but we need to use chrome.localstorage: https://github.com/localForage/localForage/issues/43

So I think it could be great to integrate this driver to ionic-storage. 

I'm working on a PR here: https://github.com/ionic-team/ionic-storage/pull/153

Let me know what you think about this!

Regards

**Comment by AndreasGassmann** [comment]
Thanks for your PR. It works very well in our project and I hope this gets merged soon.

**Comment by Nightbr** [comment]
Hope it will be merged soon as well 😁 Thanks for yout support!

**Comment by Nightbr** [comment]
any chance someone can review this? It seems there is a need from several people here.
Thanks in advance!

**Comment by AndreasGassmann** [comment]
We used this branch in our chrome extension within an ionic app, but there was a very weird error.

It was a long time ago, so I don't remember it well. But it was something about async/await not working in a specific chrome version. We ended up re-implementing part of it without async/await and that fixed the error. I'll try to remember what exactly it was and post some links.

https://github.com/bb4L/ionic-storage
